scripts/database/create_heroku_stage_database.sh

```
@@ -0,0 +1,119 @@
+        -t product_versions \
+        -t products \
+        -t raw_adi  \
+        -t raw_adi_logs \
+        -t raw_update_channels \
+        -t reasons  \
+        -t release_channel_matches \
+        -t release_channels \
+        -t release_repositories \
+        -t releases_raw  \
+        -t report_partition_info \
+        -t signature_products \
+        -t signature_products_rollup \
+        -t signatures \
+        -t skiplist \
+        -t special_product_platforms \
+        -t suspicious_crash_signatures \
+        -t transform_rules \
+        -t update_channel_map \
+        -t uptime_levels \
+        -t windows_versions \
+        -F c \
+        --no-acl --no-owner \
+        --disable-triggers \
+        -v \
+        -f ${SLIM_DUMP} \
+        ${SOURCE_DATABASE}
+
+echo date 
+
+# Move directories so that we get the alembic environment configuration
+pushd /data/socorro/application
+# Setup a fresh database from our repo
+# The hostname is set to '' so that psycopg2 uses a local socket, thereby avoiding ACLs
+$PYTHON ${VENV}/bin/socorro setupdb --database_name=${TEMP_DATABASE} --no_staticdata --dropdb --no_roles\
+	--database_hostname='' \
+	--database_superusername=postgres --database_superuserpassword="${STAGE_PASSWORD}" \
+	--database_username=postgres --database_password="${STAGE_PASSWORD}"
+popd
+
+echo date
+
+# Remove old ADI to help make restore time finite
+$PSQL ${TEMP_DATABASE} -c "DELETE FROM raw_adi where date < now() - '7 days'::interval" 
+$PSQL ${TEMP_DATABASE} -c "DELETE FROM raw_adi_logs where report_date < now() - '7 days'::interval"
+
+# Do a full dump
+pg_dump -U postgres ${TEMP_DATABASE} -F c --no-owner --no-acl -v -f ${FULL_DUMP}
+
+# Reset our stage database
+heroku pg:reset ${HEROKU_STAGE_APP} --confirm ${HEROKU_STAGE_APP}
+
+# Restore to heroku
+pg_restore --no-owner --no-acl -v -d ${HEROKU_STAGE_DB_URL} ${FULL_DUMP}
+
+# Cleanup our temp database
+$PSQL postgres -c "DROP DATABASE ${TEMP_DATABASE}"
+
+
```

Yeah, we should have set -e in there. I had an issue where I set it and then an error got swallowed though. I will investigate more closely and get `set -e` back.